cockroachcloud/quickstart-trial-cluster.md

```diff
@@ -22,7 +22,12 @@ For this tutorial, we will create a 3-node GCP cluster in the `us-west2` region.
 1. On the **Overview** page, click **Create Cluster**.
 1. On the **Create new cluster** page, for **Cloud provider**, select **Google Cloud**.
 1. For **Regions & nodes**, use the default selection of `California (us-west)` region and 3 nodes.
-1. For **Hardware per node**, select `Option 1` (2vCPU, 60 GB disk).
+1. Under **Hardware per node**, select 2vCPU for **Compute** and a 60 GB disk for **Storage**.
```

Comment:
Same as above, I would suggest 35 GiB or 75 GiB since those are part of the dropdown.